cmd/net.go

```diff
@@ -77,18 +77,58 @@ func getHostIP4(host string) (ipList set.StringSet, err error) {
 	return ipList, err
 }
 
+// byLastOctetValue implements sort.Interface used in sorting a list
+// of ip address by their last octet value.
+type byLastOctetValue []net.IP
```

Comment:
Please define the sorting order explicitly, so we don't forget the reason for sorting again - e.g. "Sort a list of IP addresses by the last octet in **descending** order". I feel this is necessary, because it is a weird (non-standard) sorting order.